go.mod

```diff
@@ -3,37 +3,37 @@ module github.com/kong/kubernetes-ingress-controller
 go 1.13
 
 require (
-	contrib.go.opencensus.io/exporter/ocagent v0.6.0 // indirect
-	contrib.go.opencensus.io/exporter/prometheus v0.1.0 // indirect
-	contrib.go.opencensus.io/exporter/stackdriver v0.12.9 // indirect
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/eapache/channels v1.1.0
 	github.com/fatih/color v1.9.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/google/go-containerregistry v0.0.0-20200131185320-aec8da010de2 // indirect
 	github.com/hashicorp/go-memdb v1.2.0 // indirect
 	github.com/hashicorp/go-uuid v1.0.1
 	github.com/kong/deck v1.2.1
 	github.com/kong/go-kong v0.13.0
 	github.com/lithammer/dedent v1.1.0
-	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a // indirect
-	github.com/mitchellh/mapstructure v1.2.2
-	github.com/openzipkin/zipkin-go v0.2.2 // indirect
-	github.com/prometheus/client_golang v1.0.0
-	github.com/sirupsen/logrus v1.4.2
+	github.com/mitchellh/mapstructure v1.3.1
+	github.com/prometheus/client_golang v1.7.1
+	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.6.2
-	github.com/stretchr/testify v1.4.0
+	github.com/spf13/viper v1.7.0
+	github.com/stretchr/testify v1.5.1
 	github.com/tidwall/gjson v1.2.1
 	github.com/tidwall/match v1.0.1 // indirect
-	github.com/tidwall/pretty v0.0.0-20190325153808-1166b9ac2b65 // indirect
-	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
+	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
 	gopkg.in/go-playground/pool.v3 v3.1.1
-	k8s.io/api v0.17.4
-	k8s.io/apimachinery v0.17.4
-	k8s.io/client-go v0.17.4
+	k8s.io/api v0.19.0-rc.4
+	k8s.io/apimachinery v0.19.0-rc.4
+	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 	k8s.io/klog v1.0.0
-	knative.dev/pkg v0.0.0-20200205160431-4ec5e09f716b
-	knative.dev/serving v0.12.1
+	knative.dev/networking v0.0.0-20200818151513-77f604177dc7
+	knative.dev/pkg v0.0.0-20200818021113-c451e3c37f5f
+)
+
+replace (
+	k8s.io/client-go => k8s.io/client-go v0.19.0-rc.4
+
+	knative.dev/networking => github.com/mflendrich/networking v0.0.0-20200820130121-b71940c64af7
+	knative.dev/pkg => github.com/mflendrich/pkg v0.0.0-20200820125340-3493cae5c655
```

Comment:
Please fork these repos under kong's account and let's use them.

Reply:
Done